shakedown/dcos/marathon.py

```diff
@@ -3,29 +3,34 @@
 
 
 def deployment_predicate(app_id=None):
-    client = marathon.create_client()
-    return len(client.get_deployments()) == 0
+    return len(marathon.create_client().get_deployments(app_id)) == 0
 
 
 def deployment_wait(timeout=120, app_id=None):
     time_wait(lambda: deployment_predicate(app_id),
               timeout)
 
 
-def delete_all_apps():
-    client = marathon.create_client()
-    client.remove_group("/")
+def delete_app(app_id, force=False):
```

Comment:
Makes sense, I had just made them `False` by default since that's what the underlying marathon library had, but I could see why we'd want to just force by default for tests: https://github.com/dcos/dcos-cli/blob/master/dcos/marathon.py#L338